qiskit/pulse/macros.py

```diff
@@ -92,6 +152,60 @@ def measure(
     return schedule
 
 
+def _measure_v2(
+    qubits: List[int],
+    target: Target,
+    meas_map: Optional[Union[List[List[int]], Dict[int, List[int]]]] = None,
+    qubit_mem_slots: Optional[Dict[int, int]] = None,
+    measure_name: str = "measure",
+) -> Schedule:
+    """Return a schedule which measures the requested qubits according to the given
+    instruction mapping and measure map, or by using the defaults provided by the backend.
+
+    By default, the measurement results for each qubit are trivially mapped to the qubit
+    index. This behavior is overridden by qubit_mem_slots. For instance, to measure
+    qubit 0 into MemorySlot(1), qubit_mem_slots can be provided as {0: 1}.
+
+    Args:
+        qubits: List of qubits to be measured.
+        backend (Union[Backend, BaseBackend]): A backend instance, which contains
+            hardware-specific data required for scheduling.
+        target: The :class:`~.Target` representing the target backend, If None, defaults to the
+                  ``target`` of ``backend``.
+        meas_map: List of sets of qubits that must be measured together. If None, defaults to
+                  the ``meas_map`` of ``backend``.
+        qubit_mem_slots: Mapping of measured qubit index to classical bit index.
+        measure_name: Name of the measurement schedule.
+
+    Returns:
+        A measurement schedule corresponding to the inputs provided.
+
+    Raises:
+        PulseError: If both ``target`` and ``backend`` is None.
+    """
+    schedule = Schedule(name=f"Default measurement schedule for qubits {qubits}")
+    if meas_map is not None:
+        if meas_map is Dict:
+            pass
+        else:
+            pass
+
+    meas_map = target.meas_map.get_qubit_groups(qubits)
```

Comment:
In _measure_v2, `meas_map` is assumed to be a List. For example, the conventional `meas_map` = [[1, 2], [2, 3, 4]] will be converted to [1, 2, 3, 4].
If `meas_map` is `None` as input, you can obtain a List like [1, 2, 3, 4] from `meas_map = target.meas_map.get_qubit_groups(qubits)`.

However, processing for Dict and List[List] types of meas_map has not been implemented yet.
I would like to know what would be the best way to handle Dict and List[List] types of meas_map.